### PR TITLE
Added docs about waterproof enclosures

### PR DESCRIPTION
**Pull request by Erol444**
image

### CODE REVIEW

File: docs/source/pages/waterproof.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+PoE devices (:ref:`OAK-D-POE <sj2088poe>` and :ref:`OAK-1-POE <sj2096>`) are shipped with an **IP67** rated enclosure. `IP code <https://en.wikipedia.org/wiki/IP_Code>`__
+tells you whether a device is dustproof and/or waterproof. IP67 means that the device is **protected from total dust ingress** and that
+it's **protected from water immersion between 15 centimeters and 1 meter in depth.** 
+
+PoE enclosures have gaskets which makes them waterproof.
+
+.. image:: /_static/images/waterproof/poe_gasket.jpeg
+
+Here's our testing of the enclosures. The heigth of this drum is 1 meter and we were testing these two enclosures over an extended period of time.
```
Comment (Luxonis-David) [inline docs/source/pages/waterproof.rst:20]: I would add that drum is filled with water

Comment (Luxonis-Brandon) [reply docs/source/pages/waterproof.rst:20]: Good idea.  Links:
https://photos.app.goo.gl/JHiYfEp3VoZgB93Z6
https://photos.app.goo.gl/jM4GhsD7mSNatGiz9

Comment (Erol444) [reply docs/source/pages/waterproof.rst:20]: @Luxonis-David good call, will do. I was wondering whether I should add the second image (the one with the meter) but I just wrote it's 1 meter, so there aren't too many images on the page.

**Comment by cafemoloko** [comment]
@Erol444, would you mind if I move the article about waterproof enclosures to _Guides_?

image

**Pull request merged**